### PR TITLE
Add Release Notes for 0.5.19

### PR DESCRIPTION
**Pull request by Caleb-Hill**
Signed-off-by: Caleb Hill <hill@bitwise.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,156 @@
...
+### splinter CLI
+
+* Fix looping database error message for insufficient file permissions to
+  sqlite databases.
+
+* Breakout list of endpoints in `splinter circuit show`. The endpoints will now
+  be listed separately: ``` Endpoints: tcp://127.0.0.1:18044
+  tcp://127.0.0.1:28044 ```
+
+## Changes in Splinter 0.5.18
```
Comment (agunde406) [inline RELEASE_NOTES.md:137]: Remove this section, it is a duplicate.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,156 @@
...
+  should be as minimal as possible.  The `InfluxRecorder` is available behind
+  the `tap` feature.
+
+* Add connection_id to `ConnectionManagerNotification::Disconnected` and
+  `ConnectionManagerNotification::NonFatalConnectionError`. This will help with
+  switching peers to be uniquely identified by the identity and the local
+  identity used.
+
+* Add picked public key to `AuthChallengeSubmitResponse`. This will allow the
+node to know which public key is being used as their identity on the other node
```
Comment (agunde406) [inline RELEASE_NOTES.md:39]: Please update the doc to the formatting is consistent, some sections do not line up with the others.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,156 @@
...
+* Move creation of accepted authorization list to the
+  `AuthProtocolRequestHandlerBuilder`. Before this was done in the handler
+  itself.
+
+*  Rename Local/Remote state machine structs to Initiating/Accepting. This will
+   hopefully make it more clear which state is for which node in the
+   authorization processes.
+
+* Add experimental commit-store feature for database backed commit hash stores.
+
```
Comment (agunde406) [inline RELEASE_NOTES.md:92]: Remove extra line

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,156 @@
...
+* Stabilize `metrics` in scabbard.
+
+### splinter CLI
+
+* Fix looping database error message for insufficient file permissions to
+  sqlite databases.
+
+* Breakout list of endpoints in `splinter circuit show`. The endpoints will now
+  be listed separately: ``` Endpoints: tcp://127.0.0.1:18044
```
Comment (agunde406) [inline RELEASE_NOTES.md:134]: The endpoint example is not formatted correctly

**Review by agunde406 — CHANGES_REQUESTED** [review]

**Review by agunde406 — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Review by rbuysse — CHANGES_REQUESTED** [review]
s/Simply/Simple

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,142 @@
...
+
+### Highlights
+
+* The experimental `metrics` feature has been renamed to `tap` and stabilized.
+
+* The use of PeerAuthorizationToken as the Peer ID has been replaced by
+  `PeerTokenPair` which is made up of both the other nodes authorization type
+  and the local authorization type used for the local node.
+
+* Simply back-pressure has been added to the scabbard `/batches` route. If
```
Comment (rbuysse) [inline RELEASE_NOTES.md:13]: s/Simply/Simple

**Review by agunde406 — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Review by rbuysse — APPROVED** [review]

**Pull request merged**